notebooks/5-stellar-flares/interm_notebook_5_classify.ipynb

```diff
@@ -0,0 +1,1428 @@
+{
```

Comment:
<p>Can we add a <code>plt.legend</code> so that we know which is which?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>